### PR TITLE
Update 10.peptide-ionization.md

### PR DESCRIPTION
**Pull request by aky**
A small intro to Ionization, MALDI and ESI.

Please provide a description of your changes:

Please check the following about your suggested changed:
- [x] The new text has one sentence per line
- [x] preprints that are cited are listed as an issue for discussion and peer review

### CODE REVIEW

**Comment by jessegmeyerlab** [comment]
Thanks for your contribution! Can you please edit so there is one sentence per line and add citations to important papers related to these topics that researchers who are new to mass spectrometry should be aware of?

File: content/10.peptide-ionization.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+Before MALDI and ESI[@DOI:10.1126/science.2675315], the so-called soft-ionization techniques were developed, the analysis of peptides was a challenge for mass spectrometry. 
+It was a big ask to introduce ionized peptides using a hard ionization technique (like fast atom bombardment) into the mass spectrometer, without destroying or breaking them.
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+Such was the impact of these two techniques that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
 
 ### MALDI
+In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]: can we cite a review of different matrices that are possible?

Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
In this method, the peptide sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like alpha-cyano-hydroxycinnamic acid).
```

File: content/10.peptide-ionization.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
 ## Peptide Ionization {.page_break_before}
-
-The 2002 Nobel Prize in Chemistry was awarded to partially to John Fenn and Koichi Tanaka "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules" [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
+Before MALDI and ESI[@DOI:10.1126/science.2675315], the so-called soft-ionization techniques were developed, the analysis of peptides was a challenge for mass spectrometry.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
Before the year 1990, peptides analysis by mass spectrometry was challenging. 
```

File: content/10.peptide-ionization.md [hunk 3]
```diff
@@ -1,7 +1,21 @@
 ## Peptide Ionization {.page_break_before}
-
-The 2002 Nobel Prize in Chemistry was awarded to partially to John Fenn and Koichi Tanaka "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules" [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
+Before MALDI and ESI[@DOI:10.1126/science.2675315], the so-called soft-ionization techniques were developed, the analysis of peptides was a challenge for mass spectrometry. 
+It was a big ask to introduce ionized peptides using a hard ionization technique (like fast atom bombardment) into the mass spectrometer, without destroying or breaking them.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
Hard ionization techniques in use at the time, like fast atom bombardment, were not directly applicable to peptides without destroying or breaking them.
```

Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]: can we cite early work on how people were trying to do FAB on peptides?

File: content/10.peptide-ionization.md (hunk 2 above)
Comment (jessegmeyerlab) [reply content/10.peptide-ionization.md]: not sure the exact year to use here?

File: content/10.peptide-ionization.md [hunk 4]
```diff
@@ -1,7 +1,21 @@
...
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+Such was the impact of these two techniques that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
```

File: content/10.peptide-ionization.md [hunk 5]
```diff
@@ -1,7 +1,21 @@
...
 
 ### MALDI
+In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).
+Subsequently, the analyte is ionized by Matrix-assisted laser desorption ionization (MALDI), in which it is subjected to a 337nm LASER.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
Subsequently, the analyte is placed in a vacuum chamber in which it is subjected to a 337nm LASER.
```

Comment (jessegmeyerlab) [reply content/10.peptide-ionization.md]: is this the only laser wavelength in use?

File: content/10.peptide-ionization.md (hunk 1 above)
Comment (jessegmeyerlab) [reply content/10.peptide-ionization.md]: could be a normal digest not necessarily in-gel

File: content/10.peptide-ionization.md [hunk 6]
```diff
@@ -1,7 +1,21 @@
...
+This laser energy is just about right to ionize the matrix and peptides without breaking them.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]: I think this sentence is trying to comment on how much energy is used? can we be more specific and include citation(s)?

File: content/10.peptide-ionization.md (hunk 1 above)
Comment (jessegmeyerlab) [reply content/10.peptide-ionization.md]: and also a general review of MALDI since you go into the mechanism here

File: content/10.peptide-ionization.md [hunk 7]
```diff
@@ -1,7 +1,21 @@
...
+The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
+The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized.
+These ionized peptides generally form the metastable ions, most of them will fragment quickly.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]: please cite

File: content/10.peptide-ionization.md [hunk 8]
```diff
@@ -1,7 +1,21 @@
...
+However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.
 
 ### Electrospray Ionization
+In the electrospray ionization, reverse-phase liquid-chromatography setup is usually connected to an online nano-ESI source.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]: please add citation(s)

File: content/10.peptide-ionization.md [hunk 9]
```diff
@@ -1,7 +1,21 @@
...
+A high voltage is applied between the spraying needle and the mass spectrometer.
+Through this needle, a fine mist of analyte as liquid droplets form at the needle.
+Due to voltage and pressure, the droplet bursts into plumes of ions carrying protons from the liquid phase of the chromatography to impart ions to the analytes.
+The liquid phase is generally kept acidic to impart protons easily to the analytes.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]: should mention taylor cone and coulombic explosion somewhere in this part, These look like a good overviews to cite: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3348530/ 
https://doi.org/10.1002/(SICI)1096-9888(199707)32:7%3C677::AID-JMS536%3E3.0.CO;2-G

File: content/12.raw-data-analysis.md [hunk 10]
```diff
@@ -61,6 +62,27 @@ The existing software for DDA and DIA analysis can be further divided into freew
 
 
 ### Analysis of DDA data
+DDA data analysis can be performed in one of the two ways- either with the vendor propreitary format directly analyzed using a propreitary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example - MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, OMSSA, or Trans-Proteomics pipeline (TPP).
+The raw data is converted to either text-based MGF (mascot generic format) or into stadard open XML implementation like mzML format.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]: cite file formats

File: content/12.raw-data-analysis.md [hunk 11]
```diff
@@ -61,6 +62,27 @@ The existing software for DDA and DIA analysis can be further divided into freew
...
+The appropriate FASTA file is chosen as a reference database to search the experimental spectra.
+All search prarameters like peptide and fragment mass errors (i.e. MS1 and MS2 tolerances), enzyme specificity, number of missed cleavages, chemical artefacts (fixed modifications) and potential biological modifications (variable/dynamic modifications) are specified before executing the search.
+The search algorithm fetches the best possible match for each query spectrum, from among a list of scored candidate peptides for that query spectrum. Each spectrum and its corresponding scored candidate peptide is called a peptide spectrum match (PSM).
+The scores reflect a _goodness-of-fit_ between an experimental spectrum and a theoretical one and do not necessarily depict the correctness of the peptide assignment.
+For evaluating the matches, a decoy database is preferred as a null model for peptide matching.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md:76]:
```suggestion

For evaluating the matches, a decoy database is preferred as a null model for peptide matching.
```

Comment (jessegmeyerlab) [reply content/12.raw-data-analysis.md:76]: new paragraph

File: content/12.raw-data-analysis.md [hunk 12]
```diff
@@ -61,6 +62,27 @@ The existing software for DDA and DIA analysis can be further divided into freew
...
+The FDR denotes the proportion of false hits in the population accepted as true.
+For Kall's method: the false hits are estimated to be the number of decoys above a given threshold. It is assumed that the number of decoy hits that pass a threshold are the false hits. A similar number of target population may also be false. Therefore, the FDR is calculated as-
+FDR = #Decoy PSMs/#Target PSMs
+
+For Elias and Gygi Method, the target population in which FDR is estimated changes.
+The target and decoy hits coming from a joint database compete against each other.
+For any spectrum, either a target or a decoy peptide can be the best hit.
+It is argued that the joint target-decoy population has decoy hits as confirmed false hits.
+However, due to the joint database search, the target database may also have equal number of false hits. Thus, the number of false hits is doubled during FDR estimation.
+FDR = (2 * # Decoy PSMs)/ (#(Target+Decoy) PSMs)
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]: please convert to latex style equation - see here for example: https://github.com/agitter/my-manuscript/blob/452ddcebf15b183dd3dec9cb5f89af027cc4daa3/content/03.delete-me.md#equations

and here:

https://www1.cmc.edu/pages/faculty/aaksoy/latex/latexthree.html

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
This is awesome thank you! Requesting some changes and additional citations. Please feel free to discuss here

File: content/10.peptide-ionization.md (hunk 5 above)
Comment (aky) [reply content/10.peptide-ionization.md]: I will need to check, this is the one used earlier. Don't know if it changed over the years

File: content/10.peptide-ionization.md (hunk 1 above)
Comment (aky) [reply content/10.peptide-ionization.md]: Agreed

**Comment by aky** [comment]
Will try to change when possible, wrote from memory, will dig up refs.

File: content/10.peptide-ionization.md (hunk 1 above)
Comment (aky) [reply content/10.peptide-ionization.md]: Do I also cover that MALDI was actually developed by Karas and Hillenkamp, but not considered for nobel, while its their  version that survives, not the one from Tanaka.

File: content/10.peptide-ionization.md (hunk 6 above)
Comment (aky) [reply content/10.peptide-ionization.md]: More than energy, its hinting towards the volatile matrix bearing the brunt of energy to dissipate their protons which effectively protects the peptides from breaking, ratio is 1000:1 :: matrix:peptides

File: content/10.peptide-ionization.md (hunk 1 above)
Comment (jessegmeyerlab) [reply content/10.peptide-ionization.md]: I think it would be great to cover that, maybe in a separate paragraph. I would prefer to discuss the controversy without taking sides since I don't know the facts well.

File: content/10.peptide-ionization.md (hunk 6 above)
Comment (jessegmeyerlab) [reply content/10.peptide-ionization.md]: I like the way you phrase this comment - can you rephrase like that?

File: content/12.raw-data-analysis.md [hunk 13]
```diff
@@ -61,6 +62,30 @@ The existing software for DDA and DIA analysis can be further divided into freew
 
 
 ### Analysis of DDA data
+DDA data analysis can be performed in one of the two ways- either with the vendor propreitary format directly analyzed using a propreitary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example - MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, OMSSA, or Trans-Proteomics pipeline (TPP).
+The raw data is converted to either text-based MGF (mascot generic format) or into stadard open XML implementation like mzML format.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
For analysis with freeware, raw data is converted to either text-based MGF (mascot generic format) or into a standard open XML format like mzML [@DOI:10.1074/mcp.R110.000133] [@PMID:20013381][@DOI:10.1074/mcp.R112.019695]. 
```

File: content/12.raw-data-analysis.md [hunk 14]
```diff
@@ -61,6 +62,30 @@ The existing software for DDA and DIA analysis can be further divided into freew
 
 
 ### Analysis of DDA data
+DDA data analysis can be performed in one of the two ways- either with the vendor propreitary format directly analyzed using a propreitary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example - MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, OMSSA, or Trans-Proteomics pipeline (TPP).
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
DDA data analysis either directly uses the vendor proprietary data format directly with a proprietary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example, MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, and OMSSA. 
Tables 1 and 4 give weblinks and citations for these software tools.
<!-- todo: need to make sure all these are listed in the table and cited -->
<!-- todo: need a paragraph discussing intergration environments like the transproteomics pipeline (TPP) and IDpicker -->
```

File: content/12.raw-data-analysis.md [hunk 15]
```diff
@@ -61,6 +62,30 @@ The existing software for DDA and DIA analysis can be further divided into freew
...
+The raw data is converted to either text-based MGF (mascot generic format) or into stadard open XML implementation like mzML format.
+The appropriate FASTA file is chosen as a reference database to search the experimental spectra.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
The appropriate FASTA file containing proteins predicted from that organism's genome is chosen as a reference database to search the experimental spectra.
```

File: content/12.raw-data-analysis.md [hunk 16]
```diff
@@ -61,6 +62,30 @@ The existing software for DDA and DIA analysis can be further divided into freew
...
+All search prarameters like peptide and fragment mass errors (i.e. MS1 and MS2 tolerances), enzyme specificity, number of missed cleavages, chemical artefacts (fixed modifications) and potential biological modifications (variable/dynamic modifications) are specified before executing the search.
+The search algorithm fetches the best possible match for each query spectrum, from among a list of scored candidate peptides for that query spectrum. Each spectrum and its corresponding scored candidate peptide is called a peptide spectrum match (PSM).
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
The search algorithm scores each query spectrum against its possible peptide matches [@DOI:10.1002/mas.21543]. 
A spectrum and its best scoring candidate peptide are called a peptide spectrum match (PSM).
```

File: content/10.peptide-ionization.md [hunk 17]
```diff
@@ -1,7 +1,21 @@
...
+Before the year 1990, peptides analysis by mass spectrometry was challenging. 
+Hard ionization techniques in use at the time, like fast atom bombardment, were not directly applicable to peptides without destroying or breaking them.
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
 
 ### MALDI
+In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).
+Subsequently, the analyte is placed in a vacuum chamber in which it is subjected to a 337nm LASER.
+This laser energy is just about right to ionize the matrix and peptides without breaking them.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
This laser energy is absorbed by the matrix, which then transfers that energy along with its free protons to the co-crystalized peptides without significantly breaking them.
```

File: content/10.peptide-ionization.md [hunk 18]
```diff
@@ -1,7 +1,21 @@
...
+Before the year 1990, peptides analysis by mass spectrometry was challenging. 
+Hard ionization techniques in use at the time, like fast atom bombardment, were not directly applicable to peptides without destroying or breaking them.
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/].
 
 ### MALDI
+In this method, the in-gel digested protein sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound (like cyno-hydroxycinnamic acid).
+Subsequently, the analyte is placed in a vacuum chamber in which it is subjected to a 337nm LASER.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
Subsequently, the analyte is placed in a vacuum chamber in which it is irradiated with a LASER, usually at 337nm [@DOI:10.1021/cr010375i].
```

File: content/12.raw-data-analysis.md [hunk 19]
```diff
@@ -61,6 +62,30 @@ The existing software for DDA and DIA analysis can be further divided into freew
...
+The FDR denotes the proportion of false hits in the population accepted as true.
+For Kall's method: the false hits are estimated to be the number of decoys above a given threshold. It is assumed that the number of decoy hits that pass a threshold are the false hits. A similar number of target population may also be false. Therefore, the FDR is calculated as-
+
+\begin{equation}FDR = #Decoy PSMs/#Target PSMs\end{equation}
+
+For Elias and Gygi Method, the target population in which FDR is estimated changes.
+The target and decoy hits coming from a joint database compete against each other.
+For any spectrum, either a target or a decoy peptide can be the best hit.
+It is argued that the joint target-decoy population has decoy hits as confirmed false hits.
+However, due to the joint database search, the target database may also have equal number of false hits. Thus, the number of false hits is doubled during FDR estimation.
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
However, due to the joint database search, the target database may also have equal number of false hits. 
Thus, the number of false hits is multiplied by two for FDR estimation.
```

File: content/12.raw-data-analysis.md [hunk 20]
```diff
@@ -61,6 +62,30 @@ The existing software for DDA and DIA analysis can be further divided into freew
...
+The scores reflect a _goodness-of-fit_ between an experimental spectrum and a theoretical one and do not necessarily depict the correctness of the peptide assignment.
+
+For evaluating the matches, a decoy database is preferred as a null model for peptide matching.
+A randomized or reversed version of target database is used as a nonparametric null model.
+The decoy database can be searched separate from the target database (Kall's method)[@DOI:10.1021/pr700600n] or it can be combined with the target database before search (Elias and Gygi method)[@DOI:10.1038/nmeth1019].
+Using either separate method or concatenated database search method, an estimate of false hits can be calculated which is used to estimate the false discovery rate (FDR) [@DOI:10.1007/978-1-4939-3106-4_7]. 
+The FDR denotes the proportion of false hits in the population accepted as true.
+For Kall's method: the false hits are estimated to be the number of decoys above a given threshold. It is assumed that the number of decoy hits that pass a threshold are the false hits. A similar number of target population may also be false. Therefore, the FDR is calculated as-
```
Comment (jessegmeyerlab) [inline content/12.raw-data-analysis.md]:
```suggestion
For Kall's method: the false hits are estimated to be the number of decoys above a given threshold. 
It is assumed that the number of decoy hits that pass a threshold are the false hits. 
A similar number of target population may also be false. 
Therefore, the FDR is calculated as:
```

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
I think the data analysis section would be done with these additional suggestions, if you can finish the address the suggestions in the ionization section we can merge this pull request. Also since this is the biggest contribution so far please feel free to move yourself to first author for now

**Comment by aky** [comment]
> I think the data analysis section would be done with these additional suggestions, if you can finish the address the suggestions in the ionization section we can merge this pull request. Also since this is the biggest contribution so far please feel free to move yourself to first author for now



> I think the data analysis section would be done with these additional suggestions, if you can finish the address the suggestions in the ionization section we can merge this pull request. Also since this is the biggest contribution so far please feel free to move yourself to first author for now

I am a bit new to this and probably going round and round figuring out how to proceed from here, there are no edit links.

**Comment by jessegmeyerlab** [comment]
> I am a bit new to this and probably going round and round figuring out how to proceed from here, there are no edit links.

The same way that you added the equations earlier today, if you make additional edits on the version of your branch https://github.com/aky/proteomics-tutorial they should get added here. Try adding a small change commit to your fork and check if it gets added to this. You would see an additional commit message in line chronologically. Please let me know if that doesn't work and I will be happy to help you figure it out. 

For the other conversations, if you agree with my suggestion you can click the 'commit changes' button below. If you want to discuss you can comment on them.

**Comment by jessegmeyerlab** [comment]
Thanks again for contributing @aky. Just wanted to follow up on the changes you proposed here and check if you need help with how to use github for writing.

Do you want to review the suggestions or should I accept them all?  You can see them by scrolling through above. If you disagree or have a different suggestion you can use the comment option to discuss: 
image

I made some suggestions about what would need to be added in the ionization section, you can see those in line as well: 
image

Please don't hesitate to let me know if you need help.

**Comment by jessegmeyerlab** [comment]
@aky here are a couple resources that you might find helpful:

1. There is a tutorial video [here](https://manubot.org/docs/getting-started.html) 
2. [this step-by-step guide](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md) from the COVID-19 review

**Comment by aky** [comment]
I am down with the virus. Will get back and have a look again, hopefully in a few days. Meanwhile, you can accept the changes that  deem fit. I can still rework the text (with citations and all) later.

**Comment by jessegmeyerlab** [comment]
> I am down with the virus. Will get back and have a look again, hopefully in a few days. Meanwhile, you can accept the changes that deem fit. I can still rework the text (with citations and all) later.

I'm so sorry to hear you are sick. Absolutely don't feel any rush from me I just wanted to check if you need any help. Please let me know if you need help when you feel better. I'm wishing for your fast recovery.

File: content/10.peptide-ionization.md (hunk 1 above)
Comment (aky) [reply content/10.peptide-ionization.md]: > can we cite a review of different matrices that are possible?

Done

Comment (aky) [reply content/10.peptide-ionization.md]: > and also a general review of MALDI since you go into the mechanism here

Added

Comment (aky) [reply content/10.peptide-ionization.md]: > I think it would be great to cover that, maybe in a separate paragraph. I would prefer to discuss the controversy without taking sides since I don't know the facts well.

Covered that in short. See if it fits.

File: content/10.peptide-ionization.md (hunk 8 above)
Comment (aky) [reply content/10.peptide-ionization.md]: added the main citation from J. Fenn.

File: content/10.peptide-ionization.md (hunk 9 above)
Comment (aky) [reply content/10.peptide-ionization.md]: Looks fine to cite these two.

File: content/10.peptide-ionization.md [hunk 21]
```diff
@@ -1,7 +1,28 @@
...
+The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale. 
+These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) "for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules". [@URL:https://www.nobelprize.org/prizes/chemistry/2002/summary/]. 
 ### MALDI
+The term, Matrix-assisted LASER desorption ionization (MALDI), was coined by Hillenkamp and Karas in 1985[@URL:https://pubs.acs.org/doi/abs/10.1021/ac00291a042]. 
+Karas and Hillenkamp discovered MALDI technique first, although a similar ionization method was shown by Koichi Tanaka in 1988 [@URL:https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/rcm.1290020802]. Few months later, Karas and Hillenkamp also demonstrated MALDI applied to protein ionization[@URL:https://pubs.acs.org/doi/abs/10.1021/ac00171a028].
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
Karas and Hillenkamp discovered the MALDI technique first, although a similar ionization method was shown by Koichi Tanaka in 1988 [@DOI:10.1002/rcm.1290020802]. 
A few months later, Karas and Hillenkamp also demonstrated MALDI applied to protein ionization [@DOI:10.1021/ac00171a028].
```

File: content/10.peptide-ionization.md [hunk 22]
```diff
@@ -1,7 +1,28 @@
...
+It also created a controversy that the widely used method of MALDI from these two people had been overlooked, and the Nobel prize was awarded to Tanaka, whose system was rarely used[@URL:https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm].
+
+In this method, the peptide sample is co-crystallized with a matrix - a volatile, low molecular-weight, aromatic compound.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
MALDI first requires the peptide sample to be co-crystallized with a matrix molecule, which is usually a volatile, low molecular-weight, organic aromatic compound.
```

File: content/10.peptide-ionization.md [hunk 23]
```diff
@@ -1,7 +1,28 @@
...
+Some examples of such compounds are cyno-hydroxycinnamic acid, dihyrobenzic acid, sinapinic acid, alpha-hydroxycinnamic acid, ferulic acid etc [@URL:https://pubmed.ncbi.nlm.nih.gov/23681820/].
+Subsequently, the analyte is placed in a vacuum chamber in which it is irradiated with a LASER, usually at 337nm [@DOI:10.1021/cr010375i].
+This laser energy is absorbed by the matrix, which then transfers that energy along with its free protons to the co-crystalized peptides without significantly breaking them.
+The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
+The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized and fly under the electrical field in the mass spectrometer.
+These ionized peptides generally form the metastable ions, most of them will fragment quickly.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
These ionized peptides generally form the metastable ions, most of them will fragment quickly [@DOI:10.1021/ac00099a029].
```

Comment (jessegmeyerlab) [reply content/10.peptide-ionization.md]: is this citation appropriate?

File: content/10.peptide-ionization.md [hunk 24]
```diff
@@ -1,7 +1,28 @@
...
+However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.
+The peptides almost always take up a single charge and thus observed and detected as [MH]+ peaks.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
Peptides ionized by MALDI almost always take up a single charge and thus observed and detected as [MH]+ species.
```

File: content/10.peptide-ionization.md [hunk 25]
```diff
@@ -1,7 +1,28 @@
...
 
 ### Electrospray Ionization
+The Electrospray Ionization (ESI) technique was developed by John Fenn and coworkers[@DOI:10.1126/science.2675315] in 1989.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
ESI was first applied to peptides by John Fenn and coworkers in 1989 [@DOI:10.1126/science.2675315].
Concepts related to electrospray Ionization (ESI) were published at least as early as 1882, when Lord Rayleigh described the number of charges that could assemble on the surface of a droplet [@DOI:10.1080/14786448208628425]. 
```

File: content/10.peptide-ionization.md [hunk 26]
```diff
@@ -1,7 +1,28 @@
...
+In the electrospray ionization, reverse-phase liquid-chromatography setup is usually connected to an online nano-ESI source.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md]:
```suggestion
ESI is usually coupled with reverse-phase liquid-chromatography of peptides directly interfaced to a mass spectrometer.
```

File: content/10.peptide-ionization.md [hunk 27]
```diff
@@ -1,7 +1,28 @@
...
+A high voltage is applied between the spraying needle and the mass spectrometer.
+Through this needle, a fine mist of analyte as liquid droplets form at the tip of the needle.
+Due to voltage and pressure, the droplet bursts into plumes of ions carrying protons from the liquid phase of the chromatography to impart ions to the analytes.
```
Comment (jessegmeyerlab) [inline content/10.peptide-ionization.md:28]:
```suggestion
Due to voltage and pressure, the droplet bursts into plumes of ions carrying protons from the liquid phase of the chromatography to impart ions to the analytes.
<!-- TODO, explicitly describe the taylor cone and coulombic explosion -->
```

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
If you are okay with these suggestions I think we can merge this into the main document

File: content/10.peptide-ionization.md (hunk 3 above)
Comment (aky) [reply content/10.peptide-ionization.md]: Going deeper into FAB may wararnt other ionization techniques explanations like electron ionization (EI), chemical ionization (CE) etc. So, I think it's better to focus on ESI and MALDI.

**Comment by aky** [comment]
> If you are okay with these suggestions I think we can merge this into the main document

Yes, changes look fine.

**Comment by jessegmeyerlab** [comment]
Thank you @aky for your contributions!!! I merged this into the main branch
